README.md

```diff
@@ -1 +1,51 @@
 Original Configuration Based On MoonEye
+
+To boot into fastboot from inital boot (Incase you cannot issue ADB commands) The following must be done:
+
+1. Hold down power until it vibrates.
+
+2. Then quickly tap the touchscreen with two fingers twice, but with each tap press the power button as well.
+
+3. It make take a few tries, but it's an easy process to master.
+
+Video form of this type of boot combo:
+
+https://www.youtube.com/watch?v=C7DH4V8kl48
+
+===============================================
+
+As of 10/25/2021, this watch is able to boot to AsteroidOS using the Android 8 (Oreo) kernel,
+
+Display, Touch and Haptics work.
+
+#TODO
```

Comment:
I think these todos are better of in separate issues (https://github.com/AsteroidOS/meta-skipjack-hybris/issues) instead of a readme.

Of course this is just a remnant from your porting work :smile: